kubernetes-examples/pom.xml

```diff
@@ -32,6 +32,8 @@
     <podName>mypod</podName>
     <masterUrl>https://localhost:8443/</masterUrl>
     <namespace>default</namespace>
+    <commons-codec.version>1.15</commons-codec.version>
+    <commons-compress.version>1.20</commons-compress.version>
```

Comment:
Shouldn't these be picked up from the parent pom??

Reply:
nice catch, I'll update it in the next update.